addons/account/models/account_move.py

```diff
@@ -5232,7 +5232,20 @@ def action_invoice_download_pdf(self):
 
     def action_print_pdf(self):
         self.ensure_one()
-        return self.env.ref('account.account_invoices').report_action(self.id)
+        report_action = self.env.ref('account.account_invoices').report_action(self.id, config=False)
+
+        if (
+            self.env.is_admin()
+            and not self.env.company.external_report_layout_id
+            and not self.env.context.get('discard_logo_check')
+        ):
+            report_action = self.env['ir.actions.report']._action_configure_external_report_layout(
+                report_action,
+                "account.action_base_document_layout_configurator",
+            )
+            report_action['context']['default_from_invoice'] = self.move_type == 'out_invoice'
+
+        return report_action
```

Comment:
Imo we could just make a function instead of copying the code from `action_invoice_sent` (and use it there too)
```suggestion
        report_action = self.env.ref('account.account_invoices').report_action(self.id, config=False)
        return self._action_with_base_document_layout_configurator(report_action)
```